src/sdk/resources/extensions.ts

```diff
@@ -36,11 +36,15 @@ export class ExtensionsResource extends Resource<Extension> implements IDownload
 		return this.getSingle(result);
 	}
 
-	async configure(id: string, values: any, params?: any, token?: CancellationToken): Promise<Extension> {
+	async updateConfiguration(id: string, values: any, params?: any, token?: CancellationToken): Promise<Extension> {
```

Comment:
This doesn't actually match up with `Extension`. It has `config_attributes` instead of `configuration`.
Not sure why it's named differently here.
```suggestion
	async updateConfiguration(id: string, values: any, params?: any, token?: CancellationToken): Promise<any> {
```

Reply:
Good call